client/txn.go

```diff
@@ -288,12 +293,18 @@ func (txn *Txn) CommitInBatch(b *Batch) error {
 
 // Commit sends an EndTransactionRequest with Commit=true.
 func (txn *Txn) Commit() error {
-	return txn.sendEndTxnCall(true /* commit */)
+	if txn.txn.Writing {
```

Comment:
It's already done, what purpose does moving the change to a separate commit provide now (other than busy work)?